py3status/module.py

```diff
@@ -182,6 +182,8 @@ def error_output(self, message, method_affected=None):
             color = color_fn(self.module_full_name, 'color_bad')
         if hasattr(color, 'none_setting'):
             color = None
+        else:
+            color = Py3()._get_color(color)
```

Comment:
This is very wrong.  We do not want to create an instance of Py3 each time we get here.  The module already has an instance of it's own if we need it but I don't think we do.

Do we know where we hit this issue?  If it is just in the config file then we can fix that when we import it.  If there are other places that we are affected then we may want to take a different approach

Reply:
Use this to slap `dpms` around.
```
diff --git a/py3status/modules/dpms.py b/py3status/modules/dpms.py
index 27345f17..8cc54dd2 100644
--- a/py3status/modules/dpms.py
+++ b/py3status/modules/dpms.py
@@ -61,6 +61,7 @@ class Py3status:
     def post_config_hook(self):
         self.color_on = self.py3.COLOR_ON or self.py3.COLOR_GOOD
         self.color_off = self.py3.COLOR_OFF or self.py3.COLOR_BAD
+        self.index = 0
 
     def dpms(self):
         """
@@ -73,6 +74,11 @@ class Py3status:
             _format = self.icon_off
             color = self.color_off
 
+        if self.index > 3:
+            self.py3.error('Oh no! That sucks.')
+
+        self.index += 1
+
         icon = self.py3.safe_format(_format)
 
         return {
```
I can't figure out what to use with...
```
from py3status.py3 import Py3, PY3_CACHE_FOREVER, ModuleErrorException
```
I see `Formatter()` does the same thing. Wrong too?